bpmsuite/bpmsuite70-businesscentral-ephemeral.json

```diff
@@ -0,0 +1,349 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "description": "Application template for Red Hat JBoss BPM Suite 7.0 which includes only a Business Central.",
+            "iconClass": "icon-jboss",
+            "tags": "bpmsuite,jboss,xpaas",
+            "version": "1.0.0",
+            "openshift.io/display-name": "Red Hat JBoss BPM Suite 7.0 Business Central",
+            "openshift.io/image.insecureRepository": "true"
+        },
+        "name": "bpmsuite70-businesscentral-ephemeral"
+    },
+    "labels": {
+        "template": "bpmsuite70-businesscentral-ephemeral",
+        "xpaas": "1.4.0"
+    },
+    "message": "A new BPM Suite Business Central application has been created in your project. The username/password for accessing the BPMS Business Central interface is \"${KIE_ADMIN_USER}/${KIE_ADMIN_PWD}\".",
+    "parameters": [
+        {
+            "displayName": "EAP Admin User",
+            "description": "EAP administrator username",
+            "name": "ADMIN_USERNAME",
+            "value": "eapadmin",
+            "required": false
+        },
+        {
+            "displayName": "EAP Admin Password",
+            "description": "EAP administrator password",
+            "name": "ADMIN_PASSWORD",
+            "from": "[a-zA-Z]{6}[0-9]{1}!",
+            "generate": "expression",
+            "required": false
+        },
+        {
+            "displayName": "KIE Admin User",
+            "description": "KIE administrator username",
+            "name": "KIE_ADMIN_USER",
+            "value": "adminUser",
+            "required": false
+        },
+        {
+            "displayName": "KIE Admin Password",
+            "description": "KIE administrator password",
+            "name": "KIE_ADMIN_PWD",
+            "from": "[a-zA-Z]{6}[0-9]{1}!",
+            "generate": "expression",
+            "required": false
+        },
+        {
+            "displayName": "KIE Server Controller User",
+            "description": "KIE server controller username (adds the user)",
+            "name": "KIE_SERVER_CONTROLLER_USER",
+            "value": "controllerUser",
+            "required": false
+        },
+        {
+            "displayName": "KIE Server Controller Password",
+            "description": "KIE server controller password (adds the user)",
+            "name": "KIE_SERVER_CONTROLLER_PWD",
+            "value": "controllerPassword",
```

Comment:
Default value set to match the one on the other template.

Reply:
I haven't been setting these (passwords), and forcing users to process the templates passing in passwords if they want them to match. Thoughts?

Reply:
I'm not sure how to share values between different application templates. It would be ideal if it was possible to define several applications / namespaces in a single template